src/main/java/org/phoebus/channelfinder/MetricsService.java

```diff
@@ -11,92 +11,99 @@
 import org.springframework.stereotype.Service;
 import org.springframework.util.LinkedMultiValueMap;
 
+import javax.annotation.PostConstruct;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 @Service
 @PropertySource(value = "classpath:application.properties")
 public class MetricsService {
 
-    private static final Logger logger = Logger.getLogger(MetricsService.class.getName());
     public static final String CF_TOTAL_CHANNEL_COUNT = "cf.total.channel.count";
     public static final String CF_PROPERTY_COUNT = "cf.property.count";
     public static final String CF_TAG_COUNT = "cf.tag.count";
     public static final String CF_CHANNEL_COUNT = "cf.channel.count";
     private static final String METRIC_DESCRIPTION_TOTAL_CHANNEL_COUNT = "Count of all ChannelFinder channels";
     private static final String METRIC_DESCRIPTION_PROPERTY_COUNT = "Count of all ChannelFinder properties";
     private static final String METRIC_DESCRIPTION_TAG_COUNT = "Count of all ChannelFinder tags";
-    private static final String METRIC_DESCRIPTION_CHANNEL_COUNT =
-            "Count of channels with specific property with and specific value";
+
     private final ChannelRepository channelRepository;
     private final PropertyRepository propertyRepository;
     private final TagRepository tagRepository;
     private final MeterRegistry meterRegistry;
 
-    MultiGauge channelCounts;
-
     @Value("${metrics.tags}")
     private String[] tags;
 
-    @Value("#{${metrics.properties:{{'pvStatus', 'Active'}, {'pvStatus', 'Inactive'}}}}")
-    private String[][] properties;
+    @Value("${metrics.properties}")
+    private String metricProperties;
+
+    Map<String, List<String>> parseProperties() {
+        if (metricProperties == null || metricProperties.isEmpty()) {
+            return new LinkedMultiValueMap<>();
+        }
+        return Arrays.stream(metricProperties.split(";")).map(s ->
```

Comment:
Might be good to give an explicit example of using this syntax; the application.properties file only has a single metricProperty.